hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/HapiSpecSetup.java

```diff
@@ -129,6 +141,14 @@ public void addOverrides(@NonNull final Map<String, String> props) {
         this.props = HapiPropertySource.inPriorityOrder(new MapPropertySource(props), this.props);
     }
 
+    /**
+     * Returns whether the default transaction memo should be the name of the {@link HapiSpec}
+     * submitting the transaction.
+     */
+    public boolean useSpecName() {
+        return props.getBoolean("memo.useSpecName");
+    }
```

Comment:
Can we just add arbitrary properties to `spec-default.properties` and use them here if we have a need for some kind of option when extending the hapispec framework?  Are there any guidelines?

Reply:
Yes pretty much if it's useful to parameterize something about the behavior of more than one spec, it is a good candidate to add something to _spec-default.properties_ imo.